### DISCUSSION TITLE
Zksync bridge insufficient ETH balance

### DISCUSSION THREAD

**Discussion by marcterra** [Cyfrin/foundry-full-course-cu#2473] [General]
Hello, 

it appears that either on Firefox or Chrome i can't bridge between ETH Sepolia Testnet and ZKsync testnet . 

I do have 0.14 ETH on Sepolia, i also have Zksync add to my wallet but i don't know why even if i put 0 it seems that there is nothing . 

How have you been able to manage it ? I know it should have been easy to do 

Regards

Marc 

![Zksync](https://github.com/user-attachments/assets/769aacb4-6078-49ee-bc5f-569464df420a)
![Capture d’écran 2024-09-14 145203](https://github.com/user-attachments/assets/fbb6be1a-9afd-4217-8a2b-a51f730af3ee)
![Capture d’écran 2024-09-14 145225](https://github.com/user-attachments/assets/05e36229-c92c-4cab-805b-6235a892c9d7)

**Comment by EngrPips** [comment]
<img width="1440" alt="Screenshot 2024-09-14 at 08 09 06" src="https://github.com/user-attachments/assets/6d370e18-7f4c-476c-a7f2-aec2dc7e9883">

I am not running into this problem; I see many people running into it. I am not sure what is wrong.
